lib/telephony/pinpoint/sms_sender.rb

```diff
@@ -30,27 +30,55 @@ def send(message:, to:)
             },
           },
         )
-        raise_if_error(response.message_response.result[to])
```

Comment:
Looks like this used to raise but now it returns an explicit error object? Does what warrant a version bump? Or is that usually done on a separate PR?)

Reply:
Yeah, I'm gonna bump the version in a different PR